src/CSS/NavBar0.module.css

```diff
@@ -0,0 +1,83 @@
+:root {
```

Comment:
I see we have NavBar.module.css inside the navigation folder an d also this NavBar0.module.css file.  Can we consolidate them down into a single file?

Reply:
Yes this